### ISSUE TITLE
npm install failure

### ISSUE THREAD

**Issue by lucker6666** [jwarby/i18n-lint#1]
➜  ~  npm install -g hslint
npm ERR! Darwin 13.1.0
npm ERR! argv "node" "/usr/local/bin/npm" "install" "-g" "hslint"
npm ERR! node v0.12.4
npm ERR! npm  v2.10.1
npm ERR! code E404

npm ERR! 404 Not Found: hslint
npm ERR! 404
npm ERR! 404 'hslint' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/lucker/npm-debug.log

**Comment by jwarby** [comment]
Hi @lucker6666, the package is nowhere near ready for release yet so I haven't published it on npm.  You can install it using:

```shell
npm install -g jwarby/hslint
```

or for the `dev` branch version, use:

```shell
npm install -g jwarby/hslint#dev
```

Hope this helps! :)

**Comment by jwarby** [comment]
Fixed in 683a061 - updated installation instructions in README
